src/components/AdminTherapist/AddPsychologist/AddPsychologist.jsx

```diff
@@ -1,11 +1,246 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import Alertify from 'alertifyjs';
+import 'alertifyjs/build/css/alertify.css';
+import { useNavigate } from 'react-router-dom';
 import styles from './AddPsychologist.module.css';
+import { addTherapist } from '../../../slices/psychologist';
 
 function AddPsychologist() {
+	const navigate = useNavigate();
+	const dispatch = useDispatch();
+	const { register, handleSubmit, formState: { errors } } = useForm();
+	const [loading, setLoading] = useState(false);
+	const forms = useRef();
+	const [form, setForm] = useState({
+		birthdate: '',
+		email: '',
+		lastName: '',
+		license: '',
+		middleName: '',
+		name: '',
+		phone: '',
+		range: '2',
+	});
+
+	const currentDate = new Date().toJSON().slice(0, 10);
+
+	const handleChange = (event) => {
+		setForm({
+			...form,
+			[event.target.id]: event.target.value,
+		});
+	};
+
+	const resetForm = () => {
+		setForm({
+			birthdate: '',
+			email: '',
+			lastName: '',
+			license: '',
+			middleName: '',
+			name: '',
+			phone: '',
+		});
+	};
+
+	const handleAdd = () => {
+		dispatch(addTherapist({ form }))
+			.unwrap()
+			.then((res) => {
+				setLoading(true);
+				if (res.isValid === true) {
+					Alertify.success(`<b style='color:white;'>Usuario Registrado 
+					</b>`);
+					setLoading(false);
+					resetForm();
+				}
+				if (res.isValid === false && res.message === 'Email already exists') {
+					Alertify.error(`<b style='color:white;'>Email existente
+					</b>`);
+					setLoading(false);
+				}
+			})
+			.catch(() => {
+				Alertify.error(`<b style='color:white;'>Error al registrar
+					</b>`);
+				setLoading(false);
+			})
+			.finally(() => {
+				setLoading(false);
+			});
+	};
 	return (
-		<div className={styles.mainContainer}>
-			<div className={styles.contForm}>
-				<h1>Psicologos</h1>
+		<div className={styles.contNewTherapist}>
+			<div className={styles.newTherapist}>
+				<h2>Registro de psicologos</h2>
```

Comment:
Registro de **psicólogos**
```suggestion
				<h2>Registro de psicólogos</h2>
```